pkg/mount/mount_unix_test.go

```diff
@@ -122,7 +137,7 @@ func TestMountReadonly(t *testing.T) {
 		}
 	}()
 
-	f, err = os.OpenFile(targetPath, os.O_RDWR, 0777)
+	_, err = os.OpenFile(targetPath, os.O_RDWR, 0777)
```

Comment:
That way you may leak an opened file descriptor, which is totally OK in this test, but I guess some other linter is going to complain.

So maybe something like
```go
if f, err := os.OpenFile(...); err == nil {
    f.Close()
￼		t.Fatal("Should not be able to open a ro file as rw")
￼	}
```

Reply:
Fixed